judge/forms.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, *args, **kwargs):
             self.fields['organizations'].queryset = Organization.objects.filter(
                 Q(is_open=True) | Q(id__in=user.profile.organizations.all()),
             )
+        if not self.fields['organizations'].queryset.exists():
```

Comment:
```suggestion
        if not self.fields['organizations'].queryset:
```
We will need the organizations later anyways, so might as well fetch them now.